razorpay-payments.php

```diff
@@ -282,23 +312,43 @@ protected function getCheckoutArguments($order, $razorpayOrderId)
               'notes'        => array(
                 'woocommerce_order_id' => $orderId
               ),
-              'order_id'     => $razorpayOrderId,
               'callback_url' => $callbackUrl,
+              'prefill' => $this->getPreFillArguments($order)
             );
 
             $args['amount'] = $this->getOrderAmountAsInteger($order);
 
+            // order_id and subscription_id both cannot be set at the same time
+            if (empty($razorpayOrderId) === false)
```

Comment:
Don't use empty for such things. Use null and set default value in method signature as null. Using empty causes type confusion.